src/Managers/FavIconManager.cs

```diff
@@ -41,10 +44,22 @@ internal static class FavIconManager {
 		/// </summary>
 		private static readonly HttpClient httpClient = new HttpClient() { Timeout = TimeSpan.FromSeconds(5) };
 
+
+		static byte[] NotFound_favico { get; set; }
+
 		public static void Init() {
 			Path.Combine(ConfigManager.AppDataPath, "FavIcons").EnsureFolderExists();
+
+			var favnf = new Bitmap(16, 16);
+			var favg = Graphics.FromImage(favnf);
+
+			favg.FillRectangle(Brushes.Beige,0,0,favnf.Width,favnf.Height);
+			favg.DrawString("na", new Font( SystemFonts.DefaultFont.FontFamily,7f),Brushes.Black,0,0);
+
+			NotFound_favico = favnf.ToByteArray(ImageFormat.Png) ;
+
```

Comment:
remove this. this icon is not required. just set the tab image to null and it will hide the icon.